frontend/src/pages/modelServing/screens/projects/ServingRuntimeTableRow.tsx

```diff
@@ -51,7 +51,7 @@ const ServingRuntimeTableRow: React.FC<ServingRuntimeTableRowProps> = ({
       : undefined;
 
   return (
-    <Tbody isExpanded={isRowExpanded}>
+    <>
       <Tr>
```

Comment:
Um -- this looks like a loss of functionality. @lucferbux 

Based on the PF docs -- https://www.patternfly.org/v4/components/table#tbody -- it looks like `Tbody` should always be true if there is an expand row 🤔 `Tr` is the one who needs `isExpanded` when things are expanded.

Reply:
Right. these row components will have two rows with one always being visible and one have isExpanded. There is no reason to have it on a tbody

```tsx
<Tr isExpanded={isRowExpanded}>
        <ServingRuntimeTableExpandedSection
          activeColumn={expandedColumn}
          obj={obj}
          onClose={() => onExpandColumn(undefined)}
          onDeployModel={onDeployModal}
        />
</Tr>
```